tests/app-data-test.json

```diff
@@ -0,0 +1,17 @@
+{
+    "webserver": {
+        "views/test/app_user": {
+            "view": "fostgres.sql",
+            "configuration": {
+                "sql": [{
+                    "path": [1, 2],
+                    "return": "object",
+                    "GET": {
+                        "command": "SELECT * FROM odin.app_user WHERE identity_id = $1 AND app_id = $2",
+                        "arguments": [1, 2]
+                    }
```

Comment:
The way you have this GET is the same as the default, so you can just use:

`"GET": "SELECT * FROM odin.app_user WHERE identity_id = $1 AND app_id = $2"`